### PR TITLE
Feature/150398711/auth screen redesign

### PR DESCRIPTION
**Pull request by 9jaswag**
#### What does this PR do?
-Fix bug on both landing and auth pages

#### Description of Task to be completed?
redesign Auth screen UI/redesign landing page UI/UX

#### How should this be manually tested?
visit the homepage and auth pages

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2095123/stories/150398708
https://www.pivotaltracker.com/story/show/150398711

### CODE REVIEW

**Comment by 9jaswag** [comment]
@andela-mpitan please check it out and merge so others can pull. Thanks

File: public/css/main.css [hunk 1]
```diff
@@ -315,42 +318,42 @@ span.auth-option {
...
+	20% { -webkit-transform: translate(-3px, 0px) rotate(1deg); }
+	30% { -webkit-transform: translate(0px, 2px) rotate(0deg); }
+	40% { -webkit-transform: translate(1px, -1px) rotate(1deg); }
+	50% { -webkit-transform: translate(-1px, 2px) rotate(-1deg); }
+	60% { -webkit-transform: translate(-3px, 1px) rotate(0deg); }
+	70% { -webkit-transform: translate(2px, 1px) rotate(-1deg); }
+	80% { -webkit-transform: translate(-1px, -1px) rotate(1deg); }
+	90% { -webkit-transform: translate(2px, 2px) rotate(0deg); }
+	100% { -webkit-transform: translate(1px, -2px) rotate(-1deg); }
 }
```
Comment (VanPaitin) [inline public/css/main.css:359]: There are indentation issues here. It should be just two spaces for indentation

File: public/css/main.css [hunk 2]
```diff
@@ -119,10 +123,9 @@ span.auth-option {
   }
 
   .foot {
-    height: 180px;
+    height: 180px !important;
```
Comment (VanPaitin) [inline public/css/main.css]: The use of `!important` as a property value is strongly discouraged in the community. Is there no other way we can achieve this without using `!important`. We might want to increase the specificity of that element and then target it accordingly.

File: public/css/main.css [hunk 3]
```diff
@@ -119,10 +123,9 @@ span.auth-option {
...
   }
 }
-
 .home-button {
     padding: 1.7rem 4.5rem;
```
Comment (VanPaitin) [inline public/css/main.css]: Please use two spaces for indentation

**Review by VanPaitin — CHANGES_REQUESTED** [review]
Please fix indentation inconsistencies and remove the !important property

File: public/css/main.css (hunk 1 above)
Comment (9jaswag) [reply public/css/main.css:359]: @andela-mpitan issues raised have been fixed

**Review by VanPaitin — APPROVED** [review]

**Pull request merged**